### PR TITLE
Update REQUIREMENTS.md for workshop 1

### PR DESCRIPTION
**Pull request by nickstenning**
I've updated the structure of the `REQUIREMENTS.md` file so that we can copy/paste it (or link to it) in an email to participants.

I've also separated out common requirements for all workshops (accounts, editor, Git, etc.) from those which may be unique to workshop 1 (Node.js).

### CODE REVIEW

**Comment by nickstenning** [comment]
*N.B.* the link to the git cheatsheet won't function until #16 is merged.

File: REQUIREMENTS.md [hunk 1]
```diff
@@ -1,25 +1,69 @@
...
-Programming languages
+   If you are familiar with another editor, feel free to use it! If you do
+   so, we can't guarantee that we'll be able to help you with it during the
+   workshop (but we'll do our best!).
 
-* Basic familiarity with **JavaScript** is required, because we’ll be developing a simple web front-end
+4. The [Git](https://git-scm.com/) distributed version control system
+   software.
 
-* Familiarity with a language chosen among **C#**, **JavaScript**, **Python** or **Java** is required for the application back-end
```
Comment (lupino3) [inline REQUIREMENTS.md:25]: I'm assuming you dropped this on purpose, and want to re-introduce closer to the appropriate workshop, right?

**Review by lupino3 — APPROVED** [review]

File: REQUIREMENTS.md (hunk 1 above)
Comment (nickstenning) [reply REQUIREMENTS.md:25]: 👍

**Comment by suzanapescador** [comment]
Looks good to me! 
Just a note that you mention installing NodeJs, which I also mention in the Setting up section of workshop 1. But, no hurt is having them twice though, hopefully by the time of the workshop they'll have it all set up :)

**Review by suzanapescador — APPROVED** [review]

**Comment by nickstenning** [comment]
Thanks @suzanapescador. Yeah, I'll tidy that up shortly :)

**Pull request merged**